External/FEXCore/Source/Interface/Core/ArchHelpers/MContext.h

```diff
@@ -18,8 +18,15 @@ enum ContextFlags : uint32_t {
   CONTEXT_FLAG_32BIT = (1U << 1),
 };
 
+#if defined(ASSERTIONS_ENABLED) && ASSERTIONS_ENABLED
+constexpr uint64_t STACK_COOKIE_MAGIC = 0x4142434445464748ULL;
+#endif
+
 struct X86ContextBackup {
   // Host State
+#if defined(ASSERTIONS_ENABLED) && ASSERTIONS_ENABLED
+  uint64_t StackCookie;
```

Comment:
Could this get a comment explaining how this is useful during signal handling?

Reply:
Added a comment.